package.json

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
Could you revert the changes in this file.

Reply:
ok